opengrok-indexer/src/main/java/org/opengrok/indexer/history/Annotation.java

```diff
@@ -24,6 +24,7 @@
 
 package org.opengrok.indexer.history;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
```

Comment:
I don't see this being used in this changeset, was it a missing import?

Reply:
yes